recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb

```diff
@@ -9,24 +9,22 @@ LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384
 
 SRC_URI = "\
     file://domu-m3ulcb.cfg \
-    file://domu-h3ulcb.cfg \
-    file://domu-salvator-x-m3.cfg \
-    file://domu-salvator-x-h3.cfg \
+    file://domu-generic-h3.cfg \
     file://domu-salvator-xs-m3n.cfg \
     file://domu-salvator-x-h3-4x2g.cfg \
     file://guest_domu \
 "
 
 S = "${WORKDIR}"
 
-DOMU_CONFIG_salvator-x-m3-xt = "domu-salvator-x-m3.cfg"
-DOMU_CONFIG_salvator-x-h3-xt = "domu-salvator-x-h3.cfg"
+DOMU_CONFIG_salvator-x-m3-xt = "domu-generic-h3.cfg"
```

Comment:
Could we choose some name without "h3" here? xxx-m3 = yyy-h3 sounds a bit confusing.

Reply:
I had same thoughts but when I noted that 'domu-salvator-x-h3-4x2g.cfg' is used for xs and h3ulcb, I decided to use '-h3' to show that m3 and h3 are equal from this point of view.
But I really OK with any name if you can propose something better.

Reply:
Names of generic configs are changed according to offline discussion:
- domu-generic-h3
- domu-generic-h3-4x2g

domu-salvator-x-m3 preserved for future modifications.